core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,1211 @@
+          <img [src]="getStaticImageUrl('/about/wifi_solid.svg')" alt="wifi icon" class="oppia-about-icon-4">
+          <span class="oppia-about-icon-text" [innerHTML]="'I18N_ABOUT_PAGE_ICON_FOUR_TEXT' | translate"></span>
+        </div>
+        <div class="oppia-about-icon-5-container">
+          <img [src]="getStaticImageUrl('/about/language_icon.svg')" alt="language icon" class="oppia-about-icon-5">
+          <span class="oppia-about-icon-text" [innerHTML]="'I18N_ABOUT_PAGE_ICON_FIVE_TEXT' | translate"></span>
         </div>
       </div>
-
-      <div class="oppia-about-extra-container">
-        <div class="oppia-static-extra-content oppia-about-clearfix">
-          <div class="oppia-about-otter-container">
-            <picture>
-              <source type="image/webp" [srcset]="getStaticImageUrl('/general/about_page_mascot.webp')">
-              <source type="image/png" [srcset]="getStaticImageUrl('/general/about_page_mascot.png')">
-              <img [src]="getStaticImageUrl('/general/about_page_mascot.png')" class="oppia-about-otter" alt="Otter">
-            </picture>
-          </div>
-          <div class="oppia-about-otter-speech-card">
-            <blockquote class="oppia-about-otter-speech-bubble">
-              <h3 [innerHTML]="'I18N_ABOUT_PAGE_SPEECH_BUBBLE' | translate"></h3>
-            </blockquote>
+    </div>
+  </div>
+  <div class="oppia-about-page-section-three">
+    <div class="oppia-section-three">
+      <h2 class="oppia-about-outcome-title-mobile" [innerHTML]="'I18N_ABOUT_PAGE_OUR_OUTCOMES' | translate"></h2>
+      <div class="oppia-about-section-three-image-1 oppia-about-section-three-right">
+        <div class="oppia-about-section-three-image">
+          <div class="oppia-about-graph-image">
+            <img [src]="getStaticImageUrl('/about/graph.svg')" alt="graph image">
           </div>
         </div>
-
-        <div class="oppia-about-extra-info oppia-about-page-extra-info">
-          <div class="oppia-static-content oppia-static-extra-content oppia-about-extra-content">
-            <div class="oppia-static-card-content oppia-static-card-content-wide oppia-about-buttons-container">
-              <a href="/community-library" class="btn oppia-about-button" [innerHTML]="'I18N_ABOUT_PAGE_LEARN_BUTTON' | translate"></a>
-              <a href="/teach" class="btn oppia-about-button" [innerHTML]="'I18N_ABOUT_PAGE_TEACH_BUTTON' | translate"></a>
-            </div>
-          </div>
+      </div>
+      <div class="oppia-about-section-three-left">
+        <h2 class="oppia-about-outcome-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_OUR_OUTCOMES' | translate"></h2>
+        <p class="oppia-about-outcome-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_OUR_OUTCOMES_CONTENT' | translate"></p>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-about-page-section-four" ng-style="{'background-image': '<[$ctrl.getCommunityBackgroundUrl()]>'}">
```

Comment:
Why do we have `$ctrl.getCommunityBackgroundUrl`? Also, why `<[` ?

Reply:
removed